scripts/neighbor_advertiser

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/env python
+
+import swsssdk
+import os
+import subprocess
+import json
+import requests
+import argparse
+
+
+DEFAULT_DURATION = 300
+NET_PATH = '/sys/class/net/'
+VLAN_PATH_PREFIX = 'Vlan'
+VLAN_ADDRESS_PATH = '/address'
+DEFAULT_DECODING = 'utf-8'
+TEMPLATE_PATH = '/usr/share/sonic/templates/neighbor_advertiser_addr.j2'
+VXLAN_CONFIG_PATH = '/tmp/neighbor_advertiser/neighbor_advertiser_vxlan_config.json'
+MIRRORING_CONFIG_NAME = 'neighbor_advertiser_mirroring_config.json'
+MIRRORING_CONFIG_PATH = '/tmp/neighbor_advertiser/' + MIRRORING_CONFIG_NAME
+FERRET_NEIGHBOR_ADVERTISER_API_PREFIX = '/Ferret/NeighborAdvertiser/Slices/'
+VXLAN_TUNNEL_NAME = 'neighbor_advertiser_tunnel'
+MIRROR_SESSION_NAME = 'neighbor_advertiser_session'
+MIRROR_TABLE_NAME = 'acl_table_mirror_na'
+MIRROR_RULE_NAME = 'acl_rule_mirror_na'
+
+
+def get_switch_name():
+    hostname = subprocess.check_output(['hostname'])
```

Comment:
No need to run `hostname`. You can find this information in the file system.
```
def read_line(filename):
    ret = ''
    with open(filename) as f:
        ret = f.read()
    return ret.strip()

hostname = read_line('/etc/hostname')
```

Reply:
Or why not socket.gethostname()?